content/en/docs/components/trainer/operator-guides/ml-policy.md

```diff
@@ -49,3 +49,9 @@ MPI-based applications. This makes it compatible with frameworks like DeepSpeed
 
 You can customize the `MPI` options such as `numProcPerNode` to define the number of slots per
 training node in the MPI hostfile.
+
+### PlainML
+
+The `PlainML` policy configures training jobs without a specialized distributed framework. It simply uses the `numNodes` value to set the job’s parallelism (number of pods) and applies any training environment variables to each pod.
+
+TrainJobs using this policy are launched as standard Kubernetes Jobs. The number of pods (parallelism) and completions is set based on the `numNodes` field in the `Trainer` spec, and environment variables from the TrainJob spec are added to the training containers.
```

Comment:
Please can you add simple example of this policy, like this:
```yaml
mlPolicy:
  numNodes: 10
```

Also, could you add move this to the first policy in the list, and add API example for MPI policy?